algorithmic_efficiency/submission_runner.py

```diff
@@ -147,7 +172,8 @@ def train_once(workload: spec.Workload, batch_size: int, data_dir: str,
                update_params: spec.UpdateParamsFn,
                data_selection: spec.DataSelectionFn,
                hyperparameters: Optional[spec.Hyperparamters],
-               rng: spec.RandomState) -> Tuple[spec.Timing, spec.Steps]:
+               rng: spec.RandomState, record: Callable,
```

Comment:
nit: can you replace the space after `rng: spec.RandomState,` with a newline?

Reply:
Done. Thanks!